recipes/dbus/1.x.x/conanfile.py

```diff
@@ -0,0 +1,115 @@
+import os
+import shutil
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = ("AFL-2.1", "GPL-2.0-or-later")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/dbus"
+    description = "D-Bus is a simple system for interprocess communication and coordination."
+    topics = ("conan", "dbus")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "with_x11": [True, False],
+        "with_glib": [True, False],
+        "enable_assert": [True, False],
+        "enable_checks": [True, False],
+        "install_system_libs": [True, False]}
```

Comment:
What system libs are installed?
Would it be useful to install them anyways (and don't have this option)?

Reply:
it's used only in https://gitlab.freedesktop.org/dbus/dbus/-/blob/dbus-1.12.20/cmake/modules/CPackInstallConfig.cmake so I'm removing the option